lib/open_project/database.rb

```diff
@@ -144,5 +144,12 @@ def self.numeric_version
         ActiveRecord::Base.connection.select_value('SHOW server_version_num;').to_i
       end
     end
+
+    # Return if the version of the underlying database engine is capable of TSVECTOR features, needed for full-text
+    # search.
+    def self.allows_tsv?
+      OpenProject::Database.name == :postgresql &&
+        Gem::Version.new(OpenProject::Database.version) >= Gem::Version.new('9.6')
```

Comment:
Do we actually need 9.6 or does 9.5 suffice? We currently have 9.5 noted down in the system requirements and it would be nice (but not necessary) if the feature would work with 9.5 as well.

Reply:
True, 9.5 or even older should work. I got confused with the phrase search that was introduced in 9.6 and our internal buzz about 9.6 (docker update etc) did not help.